### ISSUE TITLE
Jacoco html, xml report exclusion rules does not work properly on 0.6.0

### ISSUE THREAD

**Issue by k3marek** [Kotlin/kotlinx-kover#220]
After fixing #201 the jacoco report still does not work as intended.
Note - with intellij engine everything is ok.

The jacoco report contains all of the possible packages for a given module. It does not change with class filters or with html filters.

Precisely, setting the class filters excludes exact coverage values from the report which results with 0% in the report.

Setup:

```
kover {
            ...
            filters {
                classes {
                    excludes.addAll(
                         "com.companyxyz.package.*",
                          "*.di.*"
                       )
                }
            }
```
Tried also to set 
```
htmlReport {
                overrideFilters {
                    classes {
                        excludes.addAll(
                          "com.companyxyz.package.*",
                           "*.di.*"
                          )
                    }
                }
            }
```
but with no luck. The report contains all possible packages. The excluded ones shows simply 0% of coverage.
![image](https://user-images.githubusercontent.com/48361000/186448813-a3fc1eff-7f8d-4407-b4d8-8ed183727388.png)

Am I doing something wrong with the jacoco setup?

**Comment by shanshin** [comment]
Hi,
when solving issue #201, I added filtering only to the instrumentation.
Now need to add a class-file filter for the JaCoCo reporter

**Comment by shanshin** [comment]
Fixed in `0.6.1`
